romanisim/util.py

```diff
@@ -217,6 +217,8 @@ def add_more_metadata(metadata):
     metadata['exposure']['exposure_time'] = openshuttertime
     metadata['exposure']['effective_exposure_time'] = openshuttertime
     metadata['exposure']['duration'] = openshuttertime
+    read_pattern = l1.ma_table_to_tij(ma_table, read_time=1)
```

Comment:
`read_pattern` is being defined, but is not used? The next line re-calculates read_pattern and sets it directly in the meta.

Reply:
Thanks, this should be fixed now.